cmd/meroxa/root/apps/deploy.go

```diff
@@ -42,9 +42,10 @@ import (
 )
 
 const (
-	dockerHubUserNameEnv    = "DOCKER_HUB_USERNAME"
-	dockerHubAccessTokenEnv = "DOCKER_HUB_ACCESS_TOKEN" // nolint:gosec
-	pollDuration            = 2 * time.Second
+	dockerHubUserNameEnv      = "DOCKER_HUB_USERNAME"
+	dockerHubAccessTokenEnv   = "DOCKER_HUB_ACCESS_TOKEN" // nolint:gosec
+	pollDuration              = 2 * time.Second
+	requiredNumberOfResources = 2
```

Comment:
Mentioned below, but I think we should remove this check.

Reply:
Sounds good!